package.json

```diff
@@ -51,8 +51,6 @@
     "@carbon/motion": "10.5.2",
     "@carbon/themes": "10.9.3",
     "@carbon/type": "10.8.3",
-    "carbon-components": "10.9.3",
```

Comment:
- [ ] I believe these still need to remain as `dependencies` in order for us to leverage them in the codebase
- [ ] For full compatibility, we should also include the Carbon elements packages here